### PR TITLE
Update to use new run-matlab-command

### PR DESCRIPTION
**Pull request by davidbuzinski**
- Use new binary for run-command and run-tests
- skip ematlab activation if MATHWORKS_TOKEN and MATHWORKS_ACCOUNT are present in environment

### CODE REVIEW

File: src/scripts/run-tests.sh [hunk 1]
```diff
@@ -11,26 +20,36 @@ unzip -qod "${tmpdir}/scriptgen" "${tmpdir}/scriptgen.zip"
...
-    'HTMLTestReport','${PARAM_TEST_RESULTS_HTML}',\
-    'PDFTestReport','${PARAM_TEST_RESULTS_PDF}');\
-    disp('Running MATLAB script with contents:');\
-    disp(testScript.Contents);\
-    fprintf('__________\n\n');\
-    run(testScript);"
+script=command_${RANDOM}
+scriptpath=${tmpdir}/${script}.m
+echo "cd('${workdir//\'/\'\'}');" > "$scriptpath"
+cat << EOF >> "$scriptpath"
```
Comment (mcafaro) [inline src/scripts/run-tests.sh:35]: What's the idea with the move to heredoc?

Comment (davidbuzinski) [reply src/scripts/run-tests.sh:35]: It just seemed more consistent with how we run MATLAB from other tools and what we're recommending in other tools. I'm open to putting it back to how it was.

Comment (mcafaro) [reply src/scripts/run-tests.sh:35]: I think it's OK. Just don't love that we have to write another file to disk and cd around more.

Using heredoc, you should be able to get rid of the `\` at the end of each line and actually make it a multi-line script (the multi-line MATLAB commands need `...` line continuations though). Some indentation might help readability too.

File: src/scripts/run-tests.sh [hunk 2]
```diff
@@ -11,26 +20,36 @@ unzip -qod "${tmpdir}/scriptgen" "${tmpdir}/scriptgen.zip"
...
+'CoberturaModelCoverage','${PARAM_MODEL_COVERAGE_COBERTURA}',\
+'HTMLModelCoverage','${PARAM_MODEL_COVERAGE_HTML}',\
+'SimulinkTestResults','${PARAM_TEST_RESULTS_SIMULINK_TEST}',\
+'HTMLTestReport','${PARAM_TEST_RESULTS_HTML}',\
+'PDFTestReport','${PARAM_TEST_RESULTS_PDF}');\
+disp('Running MATLAB script with contents:');\
+disp(testScript.Contents);\
+fprintf('__________\n\n');\
+run(testScript);
+EOF
```
Comment (mcafaro) [inline src/scripts/run-tests.sh:53]: Untested, but think this should work?

```suggestion
cat << EOF > "$scriptpath"
cd('${workdir//\'/\'\'}');
addpath('${gendir}/scriptgen');
testScript = genscript('Test', ...
    'JUnitTestResults','${PARAM_TEST_RESULTS_JUNIT}', ...
    'CoberturaCodeCoverage','${PARAM_CODE_COVERAGE_COBERTURA}', ...
    'HTMLCodeCoverage','${PARAM_CODE_COVERAGE_HTML}', ...
    'SourceFolder','${PARAM_SOURCE_FOLDER}', ...
    'SelectByFolder','${PARAM_SELECT_BY_FOLDER}', ...
    'SelectByTag','$PARAM_SELECT_BY_TAG', ...
    'CoberturaModelCoverage','${PARAM_MODEL_COVERAGE_COBERTURA}', ...
    'HTMLModelCoverage','${PARAM_MODEL_COVERAGE_HTML}', ...
    'SimulinkTestResults','${PARAM_TEST_RESULTS_SIMULINK_TEST}', ...
    'HTMLTestReport','${PARAM_TEST_RESULTS_HTML}', ...
    'PDFTestReport','${PARAM_TEST_RESULTS_PDF}');
disp('Running MATLAB script with contents:');
disp(testScript.Contents);
fprintf('__________\n\n');
run(testScript);
EOF
```

Comment (davidbuzinski) [reply src/scripts/run-tests.sh]: Oh no, I had just committed a change removing heredoc when I saw this. I don't have a super strong difference either way. I think without heredoc is slightly more simple, but with heredoc is more consistent with our other integrations.

Comment (mcafaro) [reply src/scripts/run-tests.sh]: I could go either way as well

File: src/scripts/run-tests.sh [hunk 3]
```diff
@@ -11,12 +20,15 @@ unzip -qod "${tmpdir}/scriptgen" "${tmpdir}/scriptgen.zip"
 # form OS appropriate paths for MATLAB
 os=$(uname)
 gendir=$tmpdir
+workdir=$(pwd)
```
Comment (mcafaro) [inline src/scripts/run-tests.sh:23]: Now unused?

File: src/scripts/run-tests.sh (hunk 2 above)
Comment (davidbuzinski) [reply src/scripts/run-tests.sh]: I'm thinking maybe don't use heredoc until if/ when we need it

File: src/scripts/install.sh [hunk 4]
```diff
@@ -18,16 +18,28 @@ fi
...
+
+downloadAndRun https://ssd.mathworks.com/supportfiles/ci/ephemeral-matlab/v0/ci-install.sh --release "${PARAM_RELEASE}" $activationFlag
 
-# add MATLAB to path
 tmpdir=$(dirname "$(mktemp -u)")
 rootdir=$(cat "$tmpdir/ephemeral_matlab_root")
+
+# install matlab-batch
 if [[ $os = CYGWIN* || $os = MINGW* || $os = MSYS* ]]; then
+    batchInstallDir='C:\Program Files\matlab-batch'
```
Comment (mcafaro) [inline src/scripts/install.sh:36]:
```suggestion
    batchInstallDir='/c/Program Files/matlab-batch'
```

**Review by mcafaro — APPROVED** [review]

**Pull request merged**